templates/primary-secondary/primary-secondary.js

```diff
@@ -932,11 +943,19 @@ class TemplatePrimarySecondary extends FocusVisiblePolyfillMixin(RtlMixin(Locali
 			'd2l-template-scroll': isWindows
 		};
 		const keyboardHelpText = this._isMobile ? this.localize('templates.primary-secondary.keyboardVertical') : this.localize('templates.primary-secondary.keyboardHorizontal');
+		const primarySection = html`<main class="${classMap(scrollClasses)}"><slot name="primary"></slot></main>`;
+		const secondarySection = html`
+			<div style=${styleMap(secondaryPanelStyles)} class="d2l-template-primary-secondary-secondary-container" @transitionend=${this._onTransitionEnd}>
+				<div class="d2l-template-primary-secondary-divider-shadow"></div>
```

Comment:
This shadow only exists when in Mobile view and `secondary-first` does not have any effect in Mobile view so this would not be shown when placed in the first section.